### DISCUSSION TITLE
Question about program icon

### DISCUSSION THREAD

**Discussion by ephun** [PrismLauncher/PrismLauncher#1209] [General]
Hi there. I am switching my main launcher away from the official launcher for the first time in 13 years because I don't like the direction it's going. I was wondering if it would be possible to change the prism launcher logo on my computer to the old grass block logo. I was able to find the location that the .ico is stored in the repository, but I'm assuming it's not as cut-and-dry within the actual installation. Is this possible, and if so, how can I do it?

Really appreciate what you guys are doing.

**Comment by TheKodeToad** [comment]
I'm not sure how exactly Windows works, but I assume changing the icon of the shortcut would work?
Otherwise there are probably already tools for this ;3

**Comment by Onako2** [comment]
I made a small video how to change the icon on [YouTube](https://youtu.be/IvITZnFeRwM)
It may be a bit blurry because I just uploaded the video

**Comment by pog5** [comment]
If you want to change just the desktop icon (and the icon that appears in the taskbar, you can rightclick the Prism Launcher shortcut, go in Properties and click Change icon. From there pick the .ico file you want and click OK and OK
